Xamarin.Forms.Platform.UAP/FormsTextBox.cs

```diff
@@ -337,16 +318,13 @@ void UpdateInputScope()
 		{
 			if (IsPassword)
 			{
-				_cachedInputScope = InputScope;
 				_cachedSpellCheckSetting = IsSpellCheckEnabled;
 				_cachedPredictionsSetting = IsTextPredictionEnabled;
-				InputScope = PasswordInputScope; // Change to default input scope so we don't have suggestions, etc.
```

Comment:
Hm, as the comment says, the PasswordInputScope has a purpose. If we remove it, then whoever we added for is gonna be mad. So... we we'll need to devise a way to fix this bug without removing the PasswordInputScope.

Reply:
Looking at #4093 it seems the original author doesn't know and agrees that it seems kind of redundant with all the other disabling of text prediction going on. I don't really see any point in doing this anymore

Reply:
Nice. Will continue this discussion on that thread...